legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/preeval/tests.pure

```diff
@@ -1499,7 +1506,7 @@ function meta::pure::router::preeval::tests::getTestRuntimeWithModelConnection(s
 
 function <<access.private>> meta::pure::router::preeval::tests::assertRoundTrip(input : FunctionDefinition<Any>[1], expectedOrig : FunctionDefinition<Any>[1]):Boolean[1]
 {
-  assertRoundTrip($input, $expectedOrig, debug())

```

Comment:
As discussed offline, this is being switched off to reduce output in the automated builds (and is probably more useful to investigation, so can be switched on then)

Reply:
As discussed - this clutter logs when running thru CICD.  This to me is more a tool developers have to fix things locally, and we should avoid outputting this to github action logs.